### DISCUSSION TITLE
Improve our examples

### DISCUSSION THREAD

**Discussion by erickzanardo** [flame-engine/flame#1125] [Ideas]
Our examples are awesome, but I believe they could be even more.

Currently, our example are all focused on a specific feature from Flame, which is great to showcase them, but we could also have a few "complex" examples which would mix several features.

These example could focused on a bigger context, for example:

 - Minimal Shmup (would cover animations, sprites, control and collision)
 - Minimal top down with inventory system (sprites, animations, camera, widgets integration)
 - etc

Thoughts?

**Comment by spydon** [comment]
Sounds great, maybe we can have a special section for these things?

**Comment by luanpotter** [comment]
Yes, 100%. I was even talking with @spydon , we should have examples that use our flame_x deps as well, specially for testing their interop.

**Comment by st-pasha** [comment]
I was thinking a bit about our examples -- would it be possible to embed them directly into the documentation website (similarly how Flutter does it)? So that, for example, if you're looking at documentation for a particular class, there's an example for it right there.

As far as Markdown goes, we can write a plugin to interpret some special syntax as a placeholder for an example interstitial, and then the only tricky part is how to embed that example into an iframe in an HTML page.

**Reply by wolfenrain** [reply]
We could indeed do this using FlamePad ;D

**Reply by st-pasha** [reply]
How would this look like from the webpage perspective? For example, can I add `<script type='dart/flame'> ... code here ... </script>` into the page, and have that code automatically turn into a `FlameGame` instance?
